src/google/protobuf/unittest_well_known_types.proto

```diff
@@ -27,5 +30,83 @@ message TestWellKnownTypes {
   google.protobuf.Struct struct_field = 7;
   google.protobuf.Timestamp timestamp_field = 8;
   google.protobuf.Type type_field = 9;
-  google.protobuf.Int32Value int32_field = 10;
+  google.protobuf.DoubleValue double_field = 10;
+  google.protobuf.FloatValue float_field = 11;
+  google.protobuf.Int64Value int64_field = 12;
+  google.protobuf.UInt64Value uint64_field = 13;
+  google.protobuf.Int32Value int32_field = 14;
+  google.protobuf.UInt32Value uint32_field = 15;
+  google.protobuf.BoolValue bool_field = 16;
+  google.protobuf.StringValue string_field = 17;
+  google.protobuf.BytesValue bytes_field = 18;
+}
+
+// A repeated field for each well-known type.
```

Comment:
Just a note: we should probably point out this change once merging to the master.